### DISCUSSION TITLE
Custom actions using AutoIT

### DISCUSSION THREAD

**Discussion by Cyb3r-Monk** [cmu-sei/GHOSTS#167] [Ideas]
As I can see, AutoIT is now a part of the framework. Would it be an idea to enable user defined AutoIT scripts? Apart from that, is using AutoIT conflict with GHOSTS actions? For example, while GHOSTS creating a Word document, if an AutoIT script kicks in, does it break anything?

**Comment by sei-dupdyke** [comment]
It's kind of early days for AutoIT, so lots of possibilities here, including user scripts, but we'd want to be careful how that looks from a process perspective on the box - it should be that (ghosts.exe == the NPC) - if we move to a model where there are lots of processes that == the NPC, then i think it is hard for teams to whitelist a single process as the "user".
